website/docs/api/spangroup.md

```diff
@@ -113,13 +113,56 @@ Get a span from the group.
 > doc.spans["errors"] = [doc[0:1], doc[2:4]]
 > span = doc.spans["errors"][1]
 > assert span.text == "goi ng"
+> span.label_ = 'LABEL'
+> assert doc.spans["errors"][1] != 'LABEL' # The span within the group was not updated
 > ```
 
 | Name        | Description                           |
 | ----------- | ------------------------------------- |
 | `i`         | The item index. ~~int~~               |
 | **RETURNS** | The span at the given index. ~~Span~~ |
 
+
+## SpanGroup.\_\_setitem\_\_ {#setitem tag="method"}
```

Comment:
The new methods could get a `new="3.2.1"` marker for the new methods, and the docs file should be formatted with `prettier` (again, I know you know from previous PRs, but just making sure we don't forget it here).

Reply:
I added the `new` tag. Still haven't figured out how to make Prettier use spacy's format settings in PyCharm. I followed the plug-in instructions, but no luck - it doesn't respect settings in `.prettierrc` So no prettier for now, sorry. If someone could help with it, would be great.

Reply:
Oh, that's weird, I run `prettier` from within Pycharm. I don't think I had to edit anything specific in `.prettierrc`. But getting `prettier` to run can sometimes be a bit of a hassle. Let's not worry about it for now. If I can push to your branch directly I can look into it later.

Reply:
I just noticed that I get an error message in PyCharm's log:
```
7:38 AM	Prettier: please specify Node.js interpreter
Configure Node.js path...
```
That's probably the reason. It does format now, but it re-formats things that I haven't touched, and I can see that the lines are wider than there were originally.

I will have to take a look at some point.

Reply:
Ah, hm. I do know that there can be incompatibilities between the `node` and `npm` versions. I typically pin them at  `node` v11.15.0 & `npm` 6.14.4. Then in Pycharm you can double check your Node.js interpreter and Package manager, and Prettier package settings.

Reply:
(but again, if it's too much hassle, we'll do it for you ;-))